.gitignore

```diff
@@ -1,7 +1,6 @@
 # Node
 node_modules
 npm-debug.log
-package-lock.json
```

Comment:
🥷

This is a “ninja commit” should be done in a different PR 🙂

Reply:
Ok I made a fixup here and that will be bone by merging https://github.com/OpenTermsArchive/docs/pull/13 before.